python/doc/pyplots/P1LagrangeEvaluation.py

```diff
@@ -12,9 +12,9 @@
 cloud = ot.Cloud(mesh.getVertices(), values)
 cloud.setPointStyle("square")
 graph.add(cloud)
-graph.setColors(["blue", "red"])
 graph.setTitle("P1 Lagrange interpolation")
 
 fig = plt.figure(figsize=(10, 4))
 func_axis = fig.add_subplot(111)
-view = View(graph, figure=fig, axes=[func_axis], add_legend=False)
+View(graph, figure=fig, axes=[func_axis], add_legend=False)
```

Comment:
I think this is likely to result in an unsightly print on the HTML page, but I may be wrong

Reply:
I will check.

Reply:
I was wrong, because this is an API page illustration script, not an example script.